src/firebase_functions/private/util.py

```diff
@@ -310,6 +311,41 @@ def firebase_config() -> None | FirebaseConfig:
     return FirebaseConfig(storage_bucket=json_data.get("storageBucket"))
 
 
+def nanoseconds_timestamp_conversion(time: str) -> _dt.datetime:
+    """Converts a nanosecond timestamp and returns a datetime object of the current time in UTC"""
+
+    # Separate the date and time part from the nanoseconds.
+    datetime_str, nanosecond_str = time.replace("Z", "").split(".")
+    # Parse the date and time part of the string.
+    event_time = _dt.datetime.strptime(datetime_str, "%Y-%m-%dT%H:%M:%S")
+    # Add the microseconds and timezone.
+    event_time = event_time.replace(microsecond=int(nanosecond_str[:6]),
+                                    tzinfo=_dt.timezone.utc)
+
+    return event_time
+
+
+def is_precision_timestamp(time: str) -> bool:
+    """Return a bool which indicates if the timestamp is in nanoseconds"""
+    # Split the string into date-time and fraction of second
+    _, s_fraction = time.split(".")
+
+    # Split the fraction from the timezone specifier ('Z' or 'z')
+    s_fraction, _ = s_fraction.split(
+        "Z") if "Z" in s_fraction else s_fraction.split("z")
+
+    # If the fraction is 9 digits long, it's a nanosecond timestamp
```

Comment:
Can we change the comment to match the code below. Something like `If the fraction is more than 6 digits long, it's a nanosecond timestamp`?